fix: generate pdf v0.0.7 missing handlebars dependency

# Description

**Is this a BUG REPORT or FEATURE REQUEST ?**
BUG REPORT

**Environment :**
Node-Red

**How to reproduce it :**
Install v0.0.7 from a Node-Red dashboard and you will see the following error message:

```
23 Nov 12:11:46 - [info]  - @s1seven/node-red-generate-pdf:generate pdf : Error: Cannot find module 'handlebars'
Require stack:
- /Users/eamon/work/node-red-playground-test/node_modules/@s1seven/schema-tools-utils/dist/loaders.js
- /Users/eamon/work/node-red-playground-test/node_modules/@s1seven/schema-tools-utils/dist/index.js
- /Users/eamon/work/node-red-playground-test/node_modules/@s1seven/schema-tools-generate-pdf/dist/index.js
- /Users/eamon/work/node-red-playground-test/node_modules/@s1seven/node-red-generate-pdf/generate-pdf-node.js
- /Users/eamon/work/node-red-playground-test/node_modules/@node-red/registry/lib/loader.js
- /Users/eamon/work/node-red-playground-test/node_modules/@node-red/registry/lib/index.js
- /Users/eamon/work/node-red-playground-test/node_modules/@node-red/runtime/lib/nodes/index.js
- /Users/eamon/work/node-red-playground-test/node_modules/@node-red/runtime/lib/index.js
- /Users/eamon/work/node-red-playground-test/node_modules/node-red/lib/red.js
- /Users/eamon/work/node-red-playground-test/node_modules/node-red/red.js

```

--- a/packages/node-red-generate-pdf/package.json
+++ b/packages/node-red-generate-pdf/package.json
@@ -15,7 +15,7 @@
     }
   },
   "dependencies": {
-    "@s1seven/schema-tools-generate-pdf": "^0.2.4"
+    "@s1seven/schema-tools-generate-pdf": "^0.2.5"
   },
   "devDependencies": {
     "pdf2pic": "^2.1.4"

